src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliCompressionOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Compression
+{
+    /// <summary>
+    /// Provides compression options to be used with <see cref="BrotliStream"/>.
+    /// </summary>
+    public sealed class BrotliCompressionOptions
+    {
+        private int _quality = BrotliUtils.Quality_Default;
+
+        /// <summary>
+        /// Gets or sets the compression quality for a Brotli compression stream.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException" accessor="set">Thrown when the value is less than 0 or greater than 11.</exception>
```

Comment:
Exceptions should read as if you were to implicitly complete the sentence "The exception that is thrown when...":
```suggestion
        /// <exception cref="ArgumentOutOfRangeException" accessor="set">The value is less than 0 or greater than 11.</exception>
```